scipy/ndimage/_filters.py

```diff
@@ -306,6 +307,12 @@ def gaussian_filter(input, sigma, order=0, output=None,
         for all axes. If specified, the size of the kernel along each axis
         will be ``2*radius + 1``, and `truncate` is ignored.
         Default is None.
+    axes : tuple of int or None, optional
+        If None, it is assumed all axes are to be filtered. Otherwise, only
```

Comment:
Nit: the axes themselves will not be filtered.
```suggestion
        If None, `input` is filtered along all axes. Otherwise,
```